samples/apps/spring-petclinic/Dockerfile.app

```diff
@@ -0,0 +1,25 @@
+FROM maven:3.8.4-openjdk-11 as builder
+
+ARG PETCLINIC_REPO=https://github.com/spring-projects/spring-petclinic.git
+ARG PETCLINIC_COMMIT=ce626da705302e71771b7e22429d1cc3209e7ebb
+
+ENV PETCLINIC_DIR /workspace/petclinic.git
+
+WORKDIR ${PETCLINIC_DIR}
+
+RUN git clone ${PETCLINIC_REPO} ${PETCLINIC_DIR} && \
```

Comment:
It seems to me that the code is checked out in `${PETCLINIC_DIR}/${PETCLINIC_DIR}` ?

Reply:
Since `${PETCLINIC_DIR}` is an absolute path (it starts with `/`), `git clone` will clone into `${PETCLINIC_DIR}` (`/workspace/petclinic.git`), not `${PETCLINIC_DIR}/${PETCLINIC_DIR}`.